java-frontend/src/main/java/org/sonar/java/bytecode/cfg/BytecodeCFGBuilder.java

```diff
@@ -114,12 +114,21 @@ public boolean isVarArgs() {
     public Block exitBlock() {
       return blocks.get(0);
     }
+
+    public List<Block> blocks() {
+      return blocks;
+    }
   }
   public static class Block implements CFG.IBlock<Instruction> {
     int id;
     BytecodeCFG cfg;
     List<Instruction> instructions;
     List<Block> successors;
+
+    public String getExceptionType() {
```

Comment:
we don't have any rules about it, but I think it will be better to keep fields together and methods separated